test/SILGen/objc_init_unavailable.swift

```diff
@@ -6,7 +6,7 @@ public func callUnavailableInit(name: String) -> ClassWithUnavailableInit {
   return ClassWithUnavailableInit(bundleID: name)
 }
 
-// CHECK-LABEL: sil [ossa] @$s21objc_init_unavailable19callUnavailableInit4nameSo09ClassWitheF0CSS_tF : $@convention(thin) (@guaranteed String) -> @owned ClassWithUnavailableInit {
+// CHECK-LABEL: sil {{(\[weak_imported\])?}} [ossa] @$s21objc_init_unavailable19callUnavailableInit4nameSo09ClassWitheF0CSS_tF : $@convention(thin) (@guaranteed String) -> @owned ClassWithUnavailableInit {
```

Comment:
This seems wrong since like you said, unavailable functions are still emitted. The bug appears to be that the Float16 metadata is _not_ emitted for some reason, which is inconsistent with the behavior of an unavailable function.